models/restore_history_helpers.go

```diff
@@ -53,8 +56,15 @@ func FindRestoreHistoryItems(q *reform.Querier, filters *RestoreHistoryItemFilte
 		if _, err := FindArtifactByID(q, filters.ArtifactID); err != nil {
 			return nil, err
 		}
+
 		conditions = append(conditions, fmt.Sprintf("artifact_id = %s", q.Placeholder(idx)))
 		args = append(args, filters.ArtifactID)
+		idx++
+	}
+
+	if filters != nil && filters.Status != nil {
```

Comment:
let's move it, to one `if filters != nil` or pass it as value not pointer?

Reply:
I've tried this approach, but then linter argues that inner statements are too deep 😄

Reply:
pointer removed